config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/OutboundJmsDestinationDetail.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.fasterxml.jackson.annotation.JsonTypeName'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWd2pLY28YaV-M4wd8Bt&open=AWd2pLY28YaV-M4wd8Bt)

[ISSUE_KEY]: AWd2pLY28YaV-M4wd8Bt